### ISSUE TITLE
Research needed for: shop/hardware|Всё для дома (appears in Russia, local knowledge required)

### ISSUE THREAD

**Issue by bhousel** [osmlab/name-suggestion-index#1514]
#### :thinking: We need help deciding what to do with "shop/hardware|Всё для дома"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/hardware|Всё для дома` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/hardware|Всё для дома`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/hardware|Всё для дома`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

**Comment by ENT8R** [comment]
@Adamant36 are you sure that this is not a generic name? Google translates it to "everything for home"...
And all places I checked with Google Street View seemed to be independant of each other. Also the logo of each place contained always "Всё для дома" but it was always written in another font, so it doesn't seem to be a brand at all...

**Comment by Adamant36** [comment]
It could be generic. I figured it was something like Bed Bath & Beyond is in America. Only more generic sounding. When I looked it up there was a few brands that seemed to big enough to maybe qualify for inclusion in the index. There's a serious lack of good information on Russian area brands online though. 

Plus, Google search always seems to give me weird results I don't like when I look up Russian words. Especially in Google Images. So I tend to not research them as good as I probably could. I'll add the check local label instead so we can get local confirmation.
